.github/workflows/integration_test.yml

```diff
@@ -113,7 +113,10 @@ jobs:
         run: make e2e
         env:
           CLUSTER_PROVIDER: ${{matrix.cluster_provider}}
-          kepler_address: localhost:9102
+
+      - name: Setup tmate session
+        if: ${{ failure() }}
+        uses: mxschmitt/action-tmate@v3
```

Comment:
just a thought. Does tmate sessions expose Github secrets in any ways?

Reply:
For debugging CI. Will remove it from workflow